authority/admin/db/nosql/provisioner.go

```diff
@@ -24,6 +24,22 @@ type dbProvisioner struct {
 	SSHTemplate  *linkedca.Template        `json:"sshTemplate"`
 	CreatedAt    time.Time                 `json:"createdAt"`
 	DeletedAt    time.Time                 `json:"deletedAt"`
+	Webhooks     []dbWebhook               `json:"webhooks,omitempty"`
+}
+
+type dbBasicAuth struct {
+	Username string `json:"username"`
+	Password string `json:"password"`
+}
+
+type dbWebhook struct {
+	Name          string      `json:"name"`
+	ID            string      `json:"id"`
+	URL           string      `json:"url"`
+	Kind          string      `json:"kind"`
+	SigningSecret string      `json:"signingSecret"`
+	BearerToken   string      `json:"bearerToken"`
+	BasicAuth     dbBasicAuth `json:"basicAuth"`
```

Comment:
We can get more compact json like this:
```suggestion
	BearerToken   string      `json:"bearerToken,omitempty"`
	BasicAuth     *dbBasicAuth `json:"basicAuth,omitempty"`
```

Reply:
113d0f95